src/components/atoms/VerifiedPublisher.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { ReactElement, useEffect, useState } from 'react'
+import styles from './VerifiedPublisher.module.css'
+import { ReactComponent as VerifiedPatch } from '../../images/patch_check.svg'
+import axios from 'axios'
+import Loader from './Loader'
+import { Logger } from '@oceanprotocol/lib'
+import { useSiteMetadata } from '../../hooks/useSiteMetadata'
+
+export default function VerifiedPublisher({
+  address
+}: {
+  address: string
+}): ReactElement {
+  const [verified, setVerified] = useState(false)
+  const [loading, setLoading] = useState(false)
+
+  const { vpRegistryUri } = useSiteMetadata().appConfig
+
+  useEffect(() => {
+    if (address) {
+      setLoading(true)
+      axios
+        .get(`${vpRegistryUri}/vp/${address}/verify`)
+        .then((response) => {
+          Logger.debug('[Verification] publisher verification:', response.data)
+          setVerified(response.data?.data?.verified)
+        })
+        .finally(() => {
+          setLoading(false)
+        })
+    }
```

Comment:
Since I implemented async/await in the other places in this PR, I will revert to that and restructure the code here.

It could be a good idea to inform the user in case of server errors / unreachable API, since publishers may be verified, but the API is not responding for example.

Will have to think on some solutions to display that, but this is probably a new issue rather than something for this PR